### PR TITLE
Feature/#308 Calculate Positions Implementation for Fulfil and Reject

### PR DESCRIPTION
**Pull request by ggrg**
Functionality is added to change participants position during Fulfil Commit and Fulfil Reject.
Unit tests were added from 377 to 582.
Integration tests were added to 105.
Coverage was greatly improved.

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "SCALE": 2
   },
   "SIDECAR": {
-    "DISABLED": false,
+    "DISABLED": true,
```
Comment (rmothilal) [inline config/default.json]: please set this to false

File: src/admin/auth/index.js [hunk 2]
```diff
@@ -11,8 +11,8 @@ exports.plugin = {
     // server.auth.default('simple')
     server.auth.strategy('jwt-strategy', 'hapi-now-auth', {
       verifyJWT: true,
-      keychain: [Config.ADMIN_SECRET],
-      // keychain: ['secret'],
+      // keychain: [Config.ADMIN_SECRET],
```
Comment (rmothilal) [inline src/admin/auth/index.js]: please uncomment this

File: src/admin/auth/index.js [hunk 3]
```diff
@@ -11,8 +11,8 @@ exports.plugin = {
...
+      keychain: ['secret'],
```
Comment (rmothilal) [inline src/admin/auth/index.js]: please comment out this

**Comment by rmothilal** [comment]
Please provide a description and show the test results and coverage

**Comment by ggrg** [comment]
## Before
**Coverage summary**
Statements   : 51.61% ( 1201/2327 )
Branches     : 35.08% ( 167/476 )
Functions    : 37.02% ( 164/443 )
Lines        : 52.31% ( 1198/2290 )

**1..377**
tests 377
pass  377
**ok**

## After
**Coverage summary**
Statements   : 63.65% ( 1567/2462 )
Branches     : 53.05% ( 270/509 )
Functions    : 49.02% ( 226/461 )
Lines        : 64.16% ( 1556/2425 )

**1..582**
tests 582
pass  582

**ok**

File: config/default.json (hunk 1 above)
Comment (ggrg) [reply config/default.json]: done

File: src/admin/auth/index.js (hunk 2 above)
Comment (ggrg) [reply src/admin/auth/index.js]: done

File: src/admin/auth/index.js (hunk 3 above)
Comment (ggrg) [reply src/admin/auth/index.js]: done

File: src/handlers/transfers/handler.js [hunk 4]
```diff
@@ -264,9 +257,22 @@ const transfer = async (error, messages) => {
 
       await consumer.commitMessageSync(message)
 
+      return true
+    } else if (action.toLowerCase() === 'reject' && status.toLowerCase() === 'success') {
```
Comment (rmothilal) [inline src/handlers/transfers/handler.js]: reject and success should be constants

**Pull request merged**